time-elements.js

```diff
@@ -561,7 +563,9 @@
     var options = {
       hour: el.getAttribute('hour'),
       minute: el.getAttribute('minute'),
-      second: el.getAttribute('second')
+      second: el.getAttribute('second'),
+      timeZone: el.getAttribute('timeZone'),
+      timeZoneName: el.getAttribute('timeZoneName')
```

Comment:
> But, if we wanted to force the timezone used, we could just have the server render the timestamp in the specified timezone itself. We wouldn't need JavaScript for that.

@mislav, you are :100: correct there.

> to force the timezone used

Reading that again, I have not worded that correctly. My understanding is that it doesn't force the rendering of that timezone, but tells JavaScript additionally to the timezone string in `datetime` what the used timezone  is. A little redundant, yes.

You think we should take it out again?